### ISSUE TITLE
Display too small in 7 inch tables

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Randyfrost21/android-thomson-key-solver#59]
```
What steps will reproduce the problem?
1.open the application in portrait position 
2 . can't read full network names
3.

What is the expected output? What do you see instead?
Phone mode or better screen distributiob

What android version are you using? Use the latest version available before
entering this bug report.
Using android 4.2.2

Please provide any additional information below.

```

Original issue reported on code.google.com by `uli...@gmail.com` on 3 Apr 2013 at 11:29

**Comment by GoogleCodeExporter** [comment]
```
Yup, I have a Nexus 7 and I agree. I will fix it soon.
```

Original comment by `ruka.araujo` on 11 Apr 2013 at 9:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in commit f08fafda1461.
```

Original comment by `ruka.araujo` on 14 Apr 2013 at 8:49
- Changed state: **Fixed**
